components/Map.js

```diff
@@ -97,11 +151,33 @@ export default function Map() {
                 key={marker.id}
                 coordinate={marker.latlng}
                 title={marker.title}
-                onPress={() => Alert.alert('slider here')}
+                onPress={showProductsSwiperModal}
               />
             );
           })}
         </MapView>
+        {/* Products Swiper Modal --- start */}
+        <Portal>
+          <Modal visible={productsSwiperModalVisible} onDismiss={hideProductsSwiperModal}>
```

Comment:
in addition to hiding the swiper modal, I think it should also reset the activeTab back to 0. When another marker is clicked, the activeTab will still have the value from before, and update on snap(?)

Reply:
1. The counter is updating faster on my phone instead of the emulator. 
2. I tested out and confirmed the width is affected by the product detail card's image. Once we have set the dummy images for product detail card, it should resolve this issue.
3. It is very weird ... Map is taking longer to load on my phone than the emulator. However, any interaction with the map is faster on my phone. 
4. I reset the activeTab to 0 when the modal dismiss.